A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -321,6 +321,42 @@ var A320_Neo_UpperECAM;
                 }
             }
         }
+        getAPWarning(_time) {
+            const count = _time * 1000;
+            let apwarning;
+            if (this.getCachedSimVar("AUTOPILOT MASTER", "Bool") == false) {
+                const timer = setTimeout(function () {
```

Comment:
this variable isn't persistent across invocations of `getAPWarning`. you create a timer and then clear that same timer a few lines later. this also means that if someone clicks the AP on and off a lot there will be several timers active at once.

Reply:
ok i'll fix that mulitple AP disconnect situation

Reply:
i want to know that in JS, setTimeout still working when disconnect -> connect -> disconnect fast.
i think if i disconnect and connect fast, setTimeout is stopped working due to if. 

https://youtu.be/ZvsAmWWHnsY <- i tested the performance. i think it is okay.

should i change these code?